app/main.py

```diff
@@ -1,6 +1,33 @@
+import requests
+import os
+
+API_KEY = os.getenv("API_KEY")
+
+URL = "https://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(
+        URL,
+        params={
+            "key": API_KEY,
+            "q": CITY,
+        }
+    )
+
+    if response:
```

Comment:
Check for status code 200

Reply:
Request object allows to check status code 200 in this way

Reply:
It's better to explicitly check the status code to ensure the response was successful. This makes your code more clear and robust.